src/components/textarea/template.njk

```diff
@@ -1,7 +1,13 @@
 {% from "label/macro.njk" import govukLabel %}
 
-{{- govukLabel(classes, labelText, hintText, errorMessage, id) -}}
+{{- govukLabel({
+  "labelText": params.label.labelText,
+  "hintText": params.label.hintText,
+  "errorMessage": params.label.errorMessage,
+  "classes": params.label.classes,
```

Comment:
I like this way of referring attributes; we might want to extend this to other components as well.

Reply:
It's annoying because ideally we would just be able to do e.g. `params.label + { "id": params.id }` to merge id in, and then we wouldn't need all this repetition.

Reply:
Yes, I thought of something similar, but I couldn't find a workaround yet.
Eventually we can create a js function and make it available across templates via addGlobal.

Reply:
I considered that – I even had a custom filter working that merged attributes in – but realised that means anyone who wanted to use our templates outside of the design system would need that extra context? Not sure it's worth it.